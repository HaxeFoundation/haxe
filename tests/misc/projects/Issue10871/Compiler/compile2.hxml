--main Main
--macro Main.MacroClass.start()
--js Output.js
--no-output
--debug