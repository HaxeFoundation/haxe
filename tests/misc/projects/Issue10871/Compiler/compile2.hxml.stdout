Main.hx:16: [--main,Main,--macro,Main.MacroClass.start(),--js,Output.js,--no-output,--debug]
Main.hx:17: true
Main.hx:18: false
Main.hx:19: true
Main.hx:20: js
Main.hx:21: []
Main.hx:22: Main
Main.hx:26: sys is forbidden
Main.hx:26: python is forbidden
Main.hx:26: php is forbidden
Main.hx:26: neko is forbidden
Main.hx:26: lua is forbidden
Main.hx:26: jvm is forbidden
Main.hx:26: java is forbidden
Main.hx:26: hl is forbidden
Main.hx:26: flash is forbidden
Main.hx:26: eval is forbidden
Main.hx:26: cpp is forbidden